src/IOConnection.cpp

```diff
@@ -85,15 +86,22 @@ namespace eipScanner {
 		}
 	}
 
-	bool IOConnection::notifyTick(std::chrono::milliseconds period) {
-		auto periodInMicroS = period.count() * 1000;
+	bool IOConnection::notifyTick() {
+		auto now = std::chrono::steady_clock::now();
+		auto sinceLastHandle =
+			std::chrono::duration_cast<std::chrono::milliseconds>(now - _lastHandleTime);
+			Logger(LogLevel::TRACE) << "Last call was " << sinceLastHandle.count() << "ms ago";
```

Comment:
Unfortunately, there is an issue in our logger... This statement are calculated even if a user use a higher logging level. It could cause performance problems if you have a lot of IO connections.

Reply:
Ok, I can remove Logger call then?

Reply:
Yes, please.